compiler/tests-compiler/sys_fs.ml

```diff
@@ -50,3 +50,23 @@ let%expect_test _ =
   |};
   [%expect {|
     bbb|}]
+
+let%expect_test _ =
+  compile_and_run
+    {|
+  (match Sys.mkdir "/not/exists" 0o777 with
+  | exception Sys_error path -> print_endline ("Sys_error: " ^ path)
+  | exception _ -> print_endline "INCORRECT ERROR"
```

Comment:
Printing the exception will be more informative in the error case.

Reply:
I was just trying to debug some Windows test errors and thought the same thing 😅 Updated now.